apps/guide/content/docs/legacy/sharding/additional-information.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Additional Information
+---
+
+<Callout>This page is a follow-up and bases its code on [the previous page](./).</Callout>
```

Comment:
missing previous page url?

Reply:
no, points to `index.md` within the sharding dir